Pod/Classes/AeroRouter.m

```diff
@@ -1,47 +1,42 @@
 #import "AeroRouter.h"
 
 @interface AeroRouter()
-@property (nonatomic, readonly, copy) NSString *APIKey;
 @property (nonatomic, readonly, copy) NSURL *baseURL;
 @end
 
 @implementation AeroRouter
 
-- (instancetype)initWithAPIKey:(NSString *)APIkey baseURL:(NSURL *)baseURL
+- (instancetype)initWithBaseURL:(NSURL *)baseURL
 {
     self = [super init];
     if (!self) { return nil; }
 
-    _APIKey = APIkey;
     _baseURL = baseURL;
 
     return self;
 }
 
 - (NSURL *)urlForPath:(NSString *)path
 {
-    return [NSURL URLWithString:path relativeToURL:self.baseURL];
+    return [self.baseURL URLByAppendingPathComponent:path];
 }
 
-- (NSURLRequest *)headLastUpdateRequestForAccountID:(NSInteger)account
+- (NSURLRequest *)headLastUpdateRequest
 {
-    NSURL *url = [self urlForPath:[NSString stringWithFormat:@"/accounts/%@", @(account)]];
+    NSURL *url = [self urlForPath:[NSString stringWithFormat:@"Echo.json"]];
```

Comment:
I think we'll need some way to specify using `Echo.staging.json` here instead? Maybe we can add another parameter to `initWithBaseURL:` like `initWithBaseURL:fileName:` – happy to pair on this.

Reply:
yea, I thought of adding it in an later PR, mostly because I wanted to test how the easiest way would be to use in eigen. We can pair on it. Let's keep it outside this PR, since we also didn't have it before, so it's already an improvement.